algolia/algoliasearch-sfcc-b2c
# Lazy Loading for Prices

# Lazy Loading for Prices

Introduces lazy loading of promotional prices from SFCC instead of relying on indexed Algolia prices. This ensures real-time price accuracy and proper handling of promotional campaigns.

## Features
- New `EnablePricingLazyLoad` preference in Business Manager
- Real-time price fetching from SFCC via new `/Algolia-Price` endpoint
- Client-side price caching to minimize API calls
- Dynamic update of promotional prices and messages

## Testing
1. Enable "Pricing Lazy Load" in Business Manager
2. Create a test promotion in SFCC
3. Verify promotional prices and messages display correctly

The feature is optional and can be toggled in Business Manager settings.
